CHANGELOG.md

```diff
@@ -13,6 +64,7 @@ Ráhpajávvre is the Lule Sámi spelling of Rapaselet.
 
 - [#1246](https://github.com/nf-core/sarek/pull/1246) - Back to dev
 - [#1259](https://github.com/nf-core/sarek/pull/1259) - nf-prov plugin
+- [#1288](https://github.com/nf-core/sarek/pull/1288) - Add nf-test continuous integration.
```

Comment:
Shouldn't this be in this release?

Reply:
What do you mean? Actual integration is not implemented someone added I think the bare bones. From my side it should definitely it be in the release notes, but the autogenerated ones need to be carefully cleaned up in any case to make them readable

Reply:
yeah, but it's listed in the 3.3.2 release

Reply:
```suggestion
```